server/api_consumers/user_service.js

```diff
@@ -0,0 +1,22 @@
+const bpcClient = require('bpc_client');
+
+const US_URL = process.env.USER_SERVICE_URL;
```

Comment:
```suggestion
const USER_SERVICE_URL = process.env.USER_SERVICE_URL;
```
I would prefer this way, but I don't mind yours, so won't block it with it...